_health-care/_js/_components/error-message.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+/**
+ * An error message to be included conditionally by question components in an error state.
+ *
+ * @constructor
+ */
+class ErrorMessage extends React.Component {
+  render() {
+    return (
+      <span className="usa-input-error-message" id="input-error-message" role="alert">{this.props.message}</span>
```

Comment:
I agree. Also there's another reason why using the id from the parent component in the `ErrorMessage` id value may not work: some question components have more than one `input`, meaning they could also have more than one `ErrorMessage`. Multiple `ErrorMessage`s on the same question component would share an id, which won't work.